index.html

```diff
@@ -3327,111 +3249,6 @@ <h3><dfn>Writing or pushing content</dfn></h3>
         "`*/*json`", on the other hand, matches both.
       </div>
     </section>
-    <section> <h3>URL patterns</h3>
```

Comment:
Hmm. A lot of work is gone... :) 
Not sure it couldn't be useful at some point, but for now yes, it's to be removed.

Reply:
I am not so certain. For people using URLs being able to use * is quite handy

Reply:
How about waiting for web developers feedback?
We can easily add it if people need it.

Reply:
Sounds good.